src/components/common/needsShutdown.jsx

```diff
@@ -113,6 +117,10 @@ export function getDevicesRequiringShutdown(vm) {
     if (needsShutdownWatchdog(vm))
         devices.push(_("Watchdog"));
 
+    // SPICE
+    if (needsShutdownSpice(vm))
+        devices.push(_("SPICE"));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1387-20240122-073835-c9864bc3-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)